doc/instances.md

```diff
@@ -47,6 +47,7 @@ The currently supported keys are:
 ```{rst-class} dec-font-size break-col-1 min-width-1-15
 ```
 
+(instance-configuration)=
```

Comment:
This bit seems unrelated to the commit message "doc: move live migration content"

Reply:
I need the ID to be able to link to the options, which are part of the live migration content.